basket/news/utils.py

```diff
@@ -362,47 +362,45 @@ def get_user_data(
     """
     user = {}
 
-    if not user.get("email_id"):
-        ctms_user = None
-        try:
-            ctms_user = ctms.get(
-                token=token,
-                email=email,
-                sfdc_id=user.get("id"),
-                amo_id=amo_id,
-                fxa_id=fxa_id,
-            )
-        except CTMSNotFoundByAltIDError:
-            return None
-        except requests.exceptions.HTTPError as error:
-            if error.response.status_code == 401:
-                raise NewsletterException(
-                    "Email service provider auth failure",
-                    error_code=errors.BASKET_EMAIL_PROVIDER_AUTH_FAILURE,
-                    status_code=500,
-                )
-            else:
-                raise NewsletterException(
-                    str(error),
-                    error_code=errors.BASKET_NETWORK_FAILURE,
-                    status_code=400,
-                )
-        except CTMSNotConfigured:
+    ctms_user = None
+    try:
+        ctms_user = ctms.get(
+            token=token,
+            email=email,
+            amo_id=amo_id,
```

Comment:
don't need `amo_id` any more either right?

Reply:
Probably, I'll check out what's calling this and clean that up as well. If no AMO stuff is happening it would be nice to simplify. 👍